css/ui/style.css

```diff
@@ -105,11 +105,6 @@
     z-index: 2;
 }
 
-/* Style overrides for the ace editor */
```

Comment:
I checked, and out CSS never actually overrides their CSS, so this rule has been void for a while! It'd make the gutter beige, by the way. Doesn't seem needed. :)